crates/revm/src/handler.rs

```diff
@@ -1064,10 +1035,7 @@ where
         _evm: &mut Self::Evm,
         _exec_result: &mut <<Self::Evm as EvmTr>::Frame as FrameTr>::FrameResult,
     ) -> Result<(), Self::Error> {
-        // Pre-AllegroModerato: fee handling (refunds and queuing) done in reimburse_caller via collectFeePostTx.
-        // The actual swap and transfer to validator happens in executeBlock at the end of block processing.
-        //
-        // Post-AllegroModerato: fees are collected and swapped immediately in collectFeePreTx.
+        // Fees are collected and swapped immediately in collectFeePreTx.
```

Comment:
This comment is incorrect, fees are swapped in `reimburse_caller` which calls `collectFeePostTx`.